tests/unit/drivers/rank/aggregate/test_chunk2doc_rank_drivers.py

```diff
@@ -127,45 +139,25 @@ def create_chunk_chunk_matches_to_score():
     return Document(doc)
 
 
+@pytest.mark.parametrize('executor', [MockMaxRanker(), MockPriceDiscountRanker(), MockLengthRanker()])
 @pytest.mark.parametrize('keep_source_matches_as_chunks', [False, True])
-def test_chunk2doc_ranker_driver_mock_exec(keep_source_matches_as_chunks):
+def test_chunk2doc_ranker_driver_mock_ranker(keep_source_matches_as_chunks, executor):
     doc = create_document_to_score()
     driver = SimpleChunk2DocRankDriver(keep_source_matches_as_chunks=keep_source_matches_as_chunks)
-    executor = MockLengthRanker()
     driver.attach(executor=executor, runtime=None)
     driver._traverse_apply(DocumentSet([doc, ]))
-    assert len(doc.matches) == 4
-    assert doc.matches[0].id == '70' * 8
-    assert doc.matches[0].score.value == 7
-    assert doc.matches[1].id == '60' * 8
-    assert doc.matches[1].score.value == 6
-    assert doc.matches[2].id == '50' * 8
-    assert doc.matches[2].score.value == 5
-    assert doc.matches[3].id == '40' * 8
-    assert doc.matches[3].score.value == 4
-    for match in doc.matches:
-        # match score is computed w.r.t to doc.id
-        assert match.score.ref_id == doc.id
-        expected_chunk_matches_length = 1 if keep_source_matches_as_chunks else 0
-        assert len(match.chunks) == expected_chunk_matches_length
-
 
-@pytest.mark.parametrize('keep_source_matches_as_chunks', [False, True])
-def test_chunk2doc_ranker_driver_max_ranker(keep_source_matches_as_chunks):
-    doc = create_document_to_score()
-    driver = SimpleChunk2DocRankDriver(keep_source_matches_as_chunks=keep_source_matches_as_chunks)
-    executor = MockMaxRanker()
-    driver.attach(executor=executor, runtime=None)
-    driver._traverse_apply(DocumentSet([doc, ]))
+    scale = 1 if not isinstance(executor, MockPriceDiscountRanker) else 0.5
```

Comment:
```suggestion
    scale = 1 if not isinstance(executor, MockPriceDiscountRanker) else MockPriceDiscountRanker.discount
```

Reply:
but then the point of the test is lost I want to show how discount info arrives to the Ranker score function